app/featuresmodel.h

```diff
@@ -82,6 +90,12 @@ class FeaturesModel : public QAbstractListModel
      */
     Q_INVOKABLE QVariant convertRoleValue( const int fromRole, const QVariant &fromValue, const int toRole ) const;
 
+    /**
+     * \brief fetchingResults checks if there is a pending feature request to the layer
+     * \return TRUE if there is a pending request, FALSE if no requests are pending
+     */
+    Q_INVOKABLE bool fetchingResults() const;
```

Comment:
this does not need to be Q_INVOKABLE (it would be only needed if we wanted to use the function directly in QML - but we're using it through the property)